examples/slic_to_anaMC/job.json

```diff
@@ -8,10 +8,10 @@
         "mcTuple": "mcTuple_cfg.py",
```

Comment:
can't tag the input_files because they have not been changed, but same as always:
@cbravo135 Can you check if this file is a good example?